src/codeu/chat/common/ServerInfo.java

```diff
@@ -0,0 +1,17 @@
+
+package codeu.chat.common;
+
+import codeu.chat.util.Time;
+
+public final class ServerInfo {
+
+    public final Time startTime;
+
+    public ServerInfo() {
```

Comment:
when you have two constructors, please comment in which scenario each of the constructors is used.